doc/node-types-html/index.html.in

```diff
@@ -15,10 +15,225 @@
     <script type="text/javascript" src="js/page/Category.js"></script>
     <script type="text/javascript" src="js/page/Entry.js"></script>
     <script type="text/javascript" src="js/page/MenuItem.js"></script>
-    <script type="text/javascript" src="js/page/ExamplesView.js"></script>
-
     <!-- page scripts !-->
     <script type="text/javascript">
+        // /*
+//  * This file is part of the Soletta Project
+//  *
+//  * Copyright (C) 2015 Intel Corporation. All rights reserved.
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the "License");
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an "AS IS" BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+//  */
+(function(window){
+    function ExamplesView(){
+        /* getting the DOM element */
+        var el = $('#example');
+
+        /* setting up the back button */
+        $('.button-back').on('click',onBackClick);
+        function onBackClick(event){
+            close();
+        }
+
+        /* handling the window resize */
+        function onWindowResize(event){
+            adjustExampleView();
+        }
+
+        /* adjusting the view when the window is resized */
+        function adjustExampleView(){
+            var exampleWidth = $('#example').width() - 291-25;
+            var exampleHeight = $('#example').height()-91-32;
+
+            $('#example-image-container').css('width',exampleWidth);
+            $('#example-image-container').css('height',exampleHeight);
+        }
+
+        /* load the contents of the selected entry */
+        function loadExampleContents(_data){
+            /* loading title and description */
+            var info = $('#example-info');
+            var title = $('<h1></h1>');
+            var description = $('<p></p>');
+            title.text(_data['name']);
+            description.text(_data['description']);
+            info.append(title);
+            info.append(description);
+            // Check for local load file requests
+            var x = location.protocol;
+            var y = /^file?:/i;
+            if(y){
+                alert("Error: Running cheat-sheet from local source! \nPlease run cheat-sheet from server");
+            }
```

Comment:
This error message shouldn't be here. Let's warn the user about this error when he clicks in the copy button